roles/content_views/README.md

```diff
@@ -0,0 +1,64 @@
+theforeman.foreman.content_views
+=========================================
+
+This role creates and manages Content Views.
+
+Role Variables
+--------------
+
+This role supports the [Common Role Variables](https://github.com/theforeman/foreman-ansible-modules/blob/develop/README.md#common-role-variables).
+
+The main data structure for this role is the list of `foreman_content_views`. Each Content View requires the following fields:
+
+- `name`
+- `content_view`
+- `content_view_update`
```

Comment:
Could you include a brief description here of what type of data each field expects and how it is expected to behave? I see this affects the `version` that gets passed to the CV module but I'm not sure yet exactly how it works.